Header header

string distortion_model_rgb
string distortion_model_ir

uint32 height_rgb
uint32 width_rgb
uint32 height_ir
uint32 width_ir

float64[]   D_rgb
float64[]   D_ir

float64[9]  K_rgb
float64[9]  K_ir

float64[9]  R_rgb
float64[9]  R_ir

float64[12] P_rgb
float64[12] P_ir

float64[]   M_x_rgb
float64[]   M_y_rgb

float64     shift_d_rgb
float64     shift_m_rgb
